[ENG-12794][docs] improve programmatic access docs to make it easier to understand how to use `EXPO_TOKEN` with EAS CLI (#30502)
expo/expo: docs/pages/accounts/programmatic-access.mdx

## Access tokens usage

You can use any tokens you have created to perform actions with the EAS CLI. To use tokens, you need to define an environment variable, like `EXPO_TOKEN="token"`, before running commands.

> Once you set the `EXPO_TOKEN` environment variable, you can run any EAS CLI command being authenticated with the token without a need to run the `eas login` command.
> The `eas login` command is only used for username and password authentication. The `EXPO_TOKEN` auth method takes precedence over the username and password if both are configured.

Reviewer: ```suggestion

Once you set the `EXPO_TOKEN` environment variable, you can run any EAS CLI command authenticated with the token without running the `eas login` command. The `eas login` command is only used for username and password authentication. The `EXPO_TOKEN` auth method takes precedence over the username and password if both are configured.
```

- Let's add a new line before this paragraph.
- The first sentence was a bit hard to read. Let's improve its readability to make it a bit more clear by using active voice.
